tsconfig.json

```diff
@@ -9,10 +9,8 @@
     "allowSyntheticDefaultImports": true,
     "target": "es2017",
     "sourceMap": true,
-    "typeRoots": [
-      "node_modules/@types",
-      "src/types"
-    ]
+    "skipLibCheck": true,
```

Comment:
this is a workaround for having Nuxt as a `devDependency`, otherwise it breaks the build